content/en/blog/_posts/2022-12-06-kubernetes-1.25-blog.md

```diff
@@ -0,0 +1,190 @@
+
+#### Feature metrics are now available
+
+**Feature metrics** are now available for each Kubernetes component, making it possible to [track whether each active feature gate is enabled](https://github.com/kubernetes/kubernetes/pull/112690) by checking the component's metric endpoint for `kubernetes_feature_enabled`.
+
+#### API server tracing
+
+_FIXME: API server tracing graduates to Beta [SIG Instrumentation] need clarification, https://github.com/kubernetes/enhancements/issues/647 indicates this is not tracked for 1.26_
+
+### Removal of legacy command line arguments relating to logging
+
+Kubernetes v1.26 [removes](https://github.com/kubernetes/kubernetes/pull/112120) some command line arguments relating to logging. These command line arguments were already deprecated. For more information, see [Deprecate klog specific flags in Kubernetes Components](https://github.com/kubernetes/enhancements/tree/3cb66bd0a1ef973ebcc974f935f0ac5cba9db4b2/keps/sig-instrumentation/2845-deprecate-klog-specific-flags-in-k8s-components).
+
+### Dynamic Resource Allocation graduates to Alpha
+
+[Dynamic Resource Allocation](https://github.com/kubernetes/enhancements/blob/master/keps/sig-node/3063-dynamic-resource-allocation/README.md) is a new feature that puts resource scheduling in the hands of 3rd-party developers: it moves away from the limited "countable" interface for requesting access to resources (e.g. `nvidia.com/gpu: 2`), providing an API more akin to that of persistent volumes. Under the hood, it uses the [Container Device Interface](https://github.com/container-orchestrated-devices/container-device-interface) (CDI) to do its device injection. This feature is blocked by the `DynamicResourceAllocation` feature gate.
+
+### CEL in Admission Control graduates to Alpha [SIG API Machinery]
+
+This feature introduces a `v1alpha1` API for [validating admission policies](https://github.com/kubernetes/enhancements/issues/3488), enabling extensible admission control via [Common Expression Language](https://github.com/google/cel-spec) expressions. Currently, custom policies are enforced via [admission webhooks](https://kubernetes.io/docs/reference/access-authn-authz/extensible-admission-controllers/), which, while flexible, have a few drawbacks when compared to in-process policy enforcement. To use, enable the `ValidatingAdmissionPolicy` feature gate and the `admissionregistration.k8s.io/v1alpha1` API via `--runtime-config`.
+
+
+### Pod scheduling improvements
+
+#### `PodSchedulingReadiness` graduates to Alpha
+
+This feature introduces a `.spec.schedulingGates` field to Pod's API, to [indicate whether the Pod is allowed to be scheduled or not](https://github.com/kubernetes/enhancements/issues/3521
+). External users/controllers can use this field to hold a Pod from scheduling based on their policies and needs.
```

Comment:
Not sure if this closing parenthesis in a new line will affect to the markdown syntax.
Should we move it to the end of the previous line?

```suggestion
External users/controllers can use this field to hold a Pod from scheduling based on their policies and needs.
```

Reply:
The link is displayed correctly on the rendered version (see [preview](https://deploy-preview-38123--kubernetes-io-main-staging.netlify.app/blog/2022/12/06/kubernetes-v1-26-release/#podschedulingreadiness-graduates-to-alpha)),
but I suggest to do so because
- some of the tools (e.g., [`linkchecker.py`](https://github.com/kubernetes/website/tree/main/scripts#linkcheckerpy)) might not recognize the markdown hyperlink codes with line-breaks.
- it will not be recognized as a complete markdown hyperlink code, preventing some smart features (e.g., IntelliSense) from working.
- ...

Reply:
I will change it (the proposed commit looks like it removes the end paren instead of the newline).